logger/src/logger.cpp

```diff
@@ -0,0 +1,19 @@
+#include "logger.hpp"
+
+#include <filesystem>
+#include <iostream>
+#include <sstream>
+#include <string>
+
+namespace logging
+{
+
+void logMessage(std::string message, const std::source_location& location)
+{
+    std::ostringstream logMsg;
```

Comment:
This is a local variable so how the API is appending to the this?

Reply:
Done.. changed it to global variable